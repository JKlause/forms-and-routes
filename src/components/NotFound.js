import React from 'react';

const NotFound = () => (

  <h1>This Page Is Not Found</h1>

);

export default NotFound;
