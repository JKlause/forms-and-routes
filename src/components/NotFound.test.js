import React from 'react';
import { shallow } from 'enzyme';
import NotFound from './NotFound';

describe('NotFound component', () => {
  it('renders NotFound', () => {   
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
