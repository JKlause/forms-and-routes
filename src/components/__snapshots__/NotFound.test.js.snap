// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound component renders NotFound 1`] = `
<h1>
  This Page Is Not Found
</h1>
`;
